view/next-project/src/components/fund_information/AddModal.tsx

```diff
@@ -42,6 +43,19 @@ const OpenAddModal: FC<ModalProps> = (props) => {
     }
   }, [user, router.isReady]);
 
+  // 担当者を局でフィルタを適用
+  const [bureauId, setBureauId] = useState<number>(1);
+  const filteredUsers = useMemo(() => {
+    const filteredUsers = props.users
+      .filter((user) => {
+        return user.bureauID === bureauId;
+      })
+      .filter((user, index, self) => {
+        return self.findIndex((u) => u.name === user.name) === index;
+      });
+    return filteredUsers;
+  }, [bureauId]);
```

Comment:
このままだと、局を変えても選択したユーザがformDataの初期値のままなので、局を変えたらその局の一番上のユーザーを取得するようにしたい！
気づかなくてすまない！
（他のファイルも同様）

```suggestion
  const filteredUsers = useMemo(() => {
    const res = users
      .filter((user) => {
        return user.bureauID === bureauId;
      })
      .filter((user, index, self) => {
        return self.findIndex((u) => u.name === user.name) === index;
      });
    if(res.length !== 0) setFormData({ ...formData, userID: res[0].id });
    return res;
  }, [bureauId]);
```